src/index.ml

```diff
@@ -32,15 +32,14 @@ module Value = struct
   module String_fixed = Data.String_fixed
 end
 
-let may f = function None -> () | Some bf -> f bf
-
-let assert_and_get = function None -> assert false | Some e -> e
-
 exception RO_not_allowed
+(** Raised whenever a RO instance performs a RW-only action. *)
```

Comment:
we are using `read-write` and `read-only` in the index_intf docs, so maybe we should keep that convention.

Reply:
You mean instead of `RO` and `RW`?